src/Protocol/Protocol_1_8.h

```diff
@@ -246,6 +229,9 @@ class cProtocol_1_8_0:
 	/** Writes the block entity data for the specified block entity into the packet. */
 	virtual void WriteBlockEntity(cPacketizer & a_Pkt, const cBlockEntity & a_BlockEntity);
 
+	/** State of the protocol. 1 = status, 2 = login, 3 = game */
+	UInt32 m_State;
```

Comment:
Variable declarations go above functions. This is the way.

Reply:
I now know de wey, thank.